code/game/machinery/doors/airlock.dm

```diff
@@ -852,26 +852,24 @@ There are 9 wires.
 	return ..()
 
 /obj/machinery/door/airlock/attack_hand(mob/user as mob)
-	if(!issilicon(usr) && isElectrified() && shock(user, 100))
+	if(!issilicon(user) && isElectrified() && shock(user, 100))
 		return
 
-	// No. -- cib
-	/**
+	// Why did they comment this out this is comedy gold
 	if(ishuman(user) && prob(40) && density)
 		var/mob/living/carbon/human/H = user
 		if(H.getBrainLoss() >= 60)
```

Comment:
This should be based on brain efficiency.

Reply:
isnțt get brain loss equal to brain efficiency?

Reply:
Now it is, but if we ever get more efficient brains it won't be.